src/oscar/apps/dashboard/catalogue/tables.py

```diff
@@ -89,6 +108,21 @@ class CategoryTable(DashboardTable):
     icon = "sitemap"
     caption = ngettext_lazy("%s Category", "%s Categories")
 
+    def render_name(self, value, record):
```

Comment:
Users with view category permission should be able to view all details of category, this will result in them being able to see only the fields that are displayed in the table. Maybe show the same category change view but disable the form fields and hide the change/update button.